src/Sarif/Autogenerated/ReportingDescriptor.cs

```diff
@@ -41,87 +41,88 @@ public SarifNodeKind SarifNodeKind
         /// <summary>
         /// A stable, opaque identifier for the report.
         /// </summary>
-        [DataMember(Name = "id", IsRequired = false, EmitDefaultValue = false)]
+        [DataMember(Name = "id", IsRequired = false, EmitDefaultValue = false, Order = 1)]
         public virtual string Id { get; set; }
```

Comment:
Not sure if this is covered in oru standard yet; but we probably want to consider if we're explicitly setting an order whether we want that to be how we sort files like this or if we want to do alphabetical, etc.